thirdparty/moab/MOABConfig.new.cmake.in

```diff
@@ -50,14 +50,8 @@ set(MOAB_HAS_FBIGEOM "@ENABLE_IGEOM@")
 set(MOAB_HAS_IMESH "@ENABLE_IMESH@")
 set(MOAB_HAS_MPI "@MOAB_HAS_MPI@")
 
-if(MOAB_USE_HDF)
-  if(EXISTS "@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake")
-    include(@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake)
-  endif()
-endif()
```

Comment:
Upstream wants to know why this was removed. My guess is that HDF5's config file isn't always at this location?